funnel/views/index.py

```diff
@@ -115,3 +115,8 @@ def sw():
 @app.route('/manifest.json', methods=['GET'])
 def manifest():
     return Response(render_template('manifest.json.jinja2'), mimetype='application/json')
+
+
+@app.route('/opensearch.xml')
+def opensearch():
+    return render_template('opensearch.xml.jinja2')
```

Comment:
Since it's no longer a template, the file can move to the static folder and be served as such. Sample:

https://github.com/hasgeek/baseframe/blob/master/baseframe/views.py#L123-L145

Reply:
Another way to do this: keep it a template, but serve `{{ url_for('search', _external=true) }}?q={searchTerms}` instead of a hardcoded URL. This will help with confirming that it works in a dev environment.